packages/barrier/SPECS/barrier.spec

```diff
@@ -0,0 +1,109 @@
+%global barrier_revision 8941e56f
+Summary: Keyboard and mouse sharing solution
+Name: barrier
+Version: 2.3.3
+Release: 1%{?dist}
+License: GPLv2
+Group: System Environment/Daemons
+URL: https://github.com/debauchee/barrier
+# This source is a lie, I used 43708ae0d636539f5d9627133baba7cce3ea0812 from github
+Source0: https://github.com/debauchee/barrier/archive/barrier-v%{version}.tar.gz
+Patch100: barrier.sgifixes.patch
+
+
+BuildRequires: cmake3
+#BuildRequires: avahi-compat-libdns_sd-devel
+BuildRequires: libX11-devel
+#BuildRequires: libXtst-devel
```

Comment:
libXtst was reasonable to build and I added it to wipnonautomated, so it can be left as a build time dep here.

Reply:
Should I be running that branch on my dev machine? Curious as otherwise I don't have a copy for the obvious reason. Or should I install the single package.